### PR TITLE
fix(style): fix style of post navigation

### PR DESCRIPTION
**Pull request by Kitcham**
## Commit Message
fix(style): fix style of post navigation

## Commit Body
Fix style of post navigation for long title.  In released version, navigation will overflow the screen with a long title.

修复因标题过长而导致的文章导航样式问题。现时版本於手机等移动设备浏览时，若标题过长，导航提示可能会溢出屏幕。

## Demo
Here are some pages for demo.

此处提供两个页面以供参考。
[Kitcham's Blog - 为 ICARUS 而生的友情链接页面](https://blog.uiharu.top/archives/feat-icarus-links-page.html)
[Kitcham's Blog - 计算神经科学（一） —— 我们搞清楚自己的脑子了吗？](https://blog.uiharu.top/archives/computational-neuroscience-1-do-you-know-brain.html)

### CODE REVIEW

**Comment by Kitcham** [comment]
## ScreenShot

- Before


- After

File: include/style/article.styl [hunk 1]
```diff
@@ -148,3 +148,17 @@ article
         
         a
             color: inherit
+
+a
+    &.article-nav-prev
+        span
+            text-align: left
+            width: 100%;
```
Comment (ppoffice) [inline include/style/article.styl:156]: What about

```diff
- width: 100%
+ flex-shrink: 1
```

Before:
Screen Shot 2020-09-02 at 15 16 58


After:
Screen Shot 2020-09-02 at 15 17 29

**Review by ppoffice — CHANGES_REQUESTED** [review]

File: include/style/article.styl (hunk 1 above)
Comment (Kitcham) [reply include/style/article.styl:156]: Let me have a try

Comment (Kitcham) [reply include/style/article.styl:156]: Yes, it works perfectly.

**Comment by Kitcham** [comment]
Yes, it works perfectly. I will have a new commit here

**Comment by Kitcham** [comment]
Now new version is available on my blog. You may check it.

**Pull request merged**